foss4gna_24/.ipynb_checkpoints/big-EO-viz-checkpoint.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>Can we come up with some other filter based on data property?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Thanks. I updated to crop a subset based on bounding box. </p>